src/_start/helpers/rapidFetch.ts

```diff
@@ -5,7 +5,7 @@ export const rapidFetcher = (): Wretcher => {
   const apiKey = process.env.REACT_APP_RAPID_API_KEY;
 
   return wretch()
-    .url(`${process.env.REACT_APP_RAPID_API_ENDPOINT}/v1`)
```

Comment:
please @natachadelarosa correct your env file with `REACT_APP_RAPID_API_ENDPOINT`. It has been fix in this PR